composer.json

```diff
@@ -25,6 +25,7 @@
             "Doctrine\\DBAL\\Migrations": "lib"
         }
     },
+    "minimum-stability": "dev",
```

Comment:
Not sure this is a good idea to always use this ...
What about using [this](https://github.com/KnpLabs/KnpMenuBundle/blob/master/.travis.yml#L30) ?